lightning/src/ln/channelmanager.rs

```diff
@@ -1345,6 +1363,7 @@ where
 /// - [`FeeEstimator`] to determine transaction fee rates needed to have a transaction mined in a
 ///   timely manner
 /// - [`Router`] for finding payment paths when initiating and retrying payments
+/// - [`MessageRouter`] for finding message paths when initiating and retrying onion messages
```

Comment:
Now that `Router`, and `MessageRouter` are decoupled, should we consider renaming `Router` to `PaymentRouter`?

Reply:
Yeah I think this should be considered in a follow-up PR at least. Makes sense.